apps/integration/src/app/integrations/flyout-with-tabs/basic/flyout-demo.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component } from '@angular/core';
```

Comment:
I'm not seeing any integration tests/spec files? Was this supposed to go in the playground SPA?

Reply:
No, I need to add those! Thanks for the reminder.